stl/inc/xutility

```diff
@@ -516,7 +516,6 @@ concept _Cpp17_input_iterator = _Cpp17_iterator<_It>
 
 template <class _It>
     requires (!_Has_iter_types<_It> && _Cpp17_iterator<_It> && !_Cpp17_input_iterator<_It>
-        // Implements the proposed resolution of LWG-3283:
         && (!requires { typename _It::iterator_category; }
             || derived_from<typename _It::iterator_category, output_iterator_tag>))
 struct _Iterator_traits_base<_It> {

```

Comment:
The PR of LWG-3283 isn't in the standard.

Reply:
"I'll notice that the comment is deleted when I go to checkin and that will remind me to look at removing this."